lib/collectionspace/converter/core/exhibition.rb

```diff
@@ -10,7 +10,179 @@ def convert
+            'organizerperson' => ['organizers', 'organizer'],
+            'sponsororganization' => ['sponsors', 'sponsor'],
+            'sponsorperson' => ['sponsors', 'sponsor']
+
+          }
+          repeatstransforms = {
+            'organizerorganization' => {'authority' => ['orgauthorities', 'organization']},
+            'organizerperson' => {'authority' => ['personauthorities', 'person']},
+            'sponsororganization' => {'authority' => ['orgauthorities', 'organization']},
+            'sponsorperson' => {'authority' => ['personauthorities', 'person']}
+          }
+          CSXML::Helpers.add_repeats(xml, attributes, repeats, repeatstransforms)
+
+          # venueGroupList, venueGroup
+          venuedata = {
+            'venueorganization' => 'venue',
+            'venueplace' => 'venue',
+            'venuestoragelocation' => 'venue',
+            'venueopeningdate' => 'venueOpeningDate',
+            'venueclosingdate' => 'venueClosingDate',
+            'venueattendance' => 'venueAttendance',
+            'venueurl' => 'venueUrl'
+          }
+
+          venuetransforms = {
+            'venueorganization' => {'authority' => ['orgauthorities', 'organization']},
+            'venueplace' => {'authority' => ['placeauthorities', 'place']},
+            'venuestoragelocation' => {'authority' => ['locationauthorities', 'location']},
+            'venueopeningdate' => {'special' => 'unstructured_date_stamp'},
+            'venueclosingdate' => {'special' => 'unstructured_date_stamp'},
+          }
+          CSXML.add_single_level_group_list(
+            xml, attributes,
+            'venue',
+            venuedata,
+            venuetransforms
+          )
+
+          # workingGroupList, workingGroup, exhibitionPersonGroupList, exhibitionPersonGroup
+          workingdata = {
+            'workinggroupnote' => 'workingGroupNote',
+            'workinggrouptitle' => 'workingGroupTitle',
+            'exhibitionpersonrole' => 'exhibitionPersonRole',
+            'exhibitionpersonorganization' => 'exhibitionPerson',
+            'exhibitionpersonperson' => 'exhibitionPerson'
+          }
+          exhibitionpersondata = [
+            'exhibitionPersonRole',
+            'exhibitionPerson'
+          ]
+          eptransforms = {
+            'exhibitionpersonorganization' => {'authority' => ['orgauthorities', 'organization']},
+            'exhibitionpersonperson' => {'authority' => ['personauthorities', 'person']},
+            'exhibitionpersonrole' => {'vocab' => 'exhibitionpersonrole'}
+          }
+          CSXML.add_nested_group_lists(
+            xml, attributes,
+            'working',
+            workingdata,
+            'exhibitionPerson',
+            exhibitionpersondata,
+            eptransforms,
+            sublist_suffix: 'GroupList',
+            subgroup_suffix: 'Group'
+          )
+
+          # galleryRotationGroupList, galleryRotationGroup
```

Comment:
@Paulmulonzia - Preferred practice: add comments with the names of the parent fields that will be created by each section. This means, in the future, if you need to quickly get to the code that created "galleryRotationGroup", you can search Github repo for it and easily find it. 

It also clearly breaks up the code visually

Reply:
Sure, I will ensure this is enforced.